src/main/java/org/folio/gobi/HelperUtils.java

```diff
@@ -6,11 +6,10 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.CompletionException;
-
 import org.folio.gobi.exceptions.HttpException;
-
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
```

Comment:
Just to be consistent can we use `import io.vertx.core.logging.LoggerFactory;`
`private static final Logger logger = LoggerFactory.getLogger(PostGobiOrdersHelper.class);`
?

Reply:
The idea behind not having Utils with its own logger is that the class where these helper Methods are called will be present in the stacktrace